integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -1153,6 +1161,244 @@ private static String getForwardedPort(String portForwardFileName) {
     return forwardedPortNo;
   }
 
+  /**
+   * Start a port-forward process and tests using forwarded port.
+   *
+   * @param domainUid domain uid
+   * @param domainNamespace domain namespace
+   * @param istioIngressPort istio ingress port
+   * @return generated local forward port
+   */
+  public static int portForwardingTests(String domainUid,
+                                        String domainNamespace,
+                                        int istioIngressPort) {
+    LoggingFacade logger = getLogger();
+
+    // verify WebLogic console is accessible before port forwarding using ingress port
+    String consoleUrl = "http://" + K8S_NODEPORT_HOST + ":" + istioIngressPort + "/console/login/LoginForm.jsp";
+    boolean checkConsole = checkAppUsingHostHeader(consoleUrl, domainNamespace + ".org");
+    assertTrue(checkConsole, "Failed to access WebLogic console");
+    logger.info("WebLogic console is accessible");
+
+    // forwarding admin port to a local port
+    String localhost = "localhost";
+    String forwardedPort = startPortForwardProcess(localhost, domainNamespace, domainUid, 7001);
+    assertNotNull(forwardedPort, "port-forward command fails to assign local port");
+    logger.info("Forwarded local port is {0}", forwardedPort);
+
+    // verify WebLogic console is accessible after port forwarding using the forwarded port
+    consoleUrl = "http://" + localhost + ":" + forwardedPort + "/console/login/LoginForm.jsp";
+    checkConsole = checkAppUsingHostHeader(consoleUrl, domainNamespace + ".org");
+    assertTrue(checkConsole, "Failed to access WebLogic console thru port-forwarded port");
+    logger.info("WebLogic console is accessible thru port forwarding");
+
+    // test accessing WLS vis WLST using the forwarded port.
+    ExecResult result = accesseWLSViaWLSUsingForwardedPort(domainUid, domainNamespace, forwardedPort);
+    assertNotNull(result, "Connecting to WebLogic failed");
+    logger.info("Connecting to Weblogic via WLST using forwarded port {0} returned {1}", result.toString());
+    assertTrue(result.stdout().contains("Successfully connected to Admin Server"),
+        "Failed to connect to WebLogic via WLST using forwarded port");
+
+    // stop port forwarding process
+    stopPortForwardProcess(domainNamespace);
+
+    return Integer.parseInt(forwardedPort);
+  }
+
+  /**
+   * Access WLS vis WLST using the forwarded port.
+   *
+   * @param domainUid domain uid
+   * @param domainNamespace domain namespace
+   * @param forwardedPort forwarded local port number to access WebLogic
+   * @return ExecResult output of executing WLST script
+   */
+  public static ExecResult accesseWLSViaWLSUsingForwardedPort(String domainUid,
```

Comment:
I do not think, this is a common utility method to be used by multiple class.  Can me move the method to ItIstioMiiDomain.java

Reply:
yea, this method is related to use WLST test case only. I put this method in CommonTestUtils.java because:

1. WLST test case is currently in IstioMII domain, We may want to add to other domain type later, if we do, we don't want to have this method everywhere, do we
2. is we move accesseWLSViaWLSUsingForwardedPort to ItIstioMiiDomain.java, we better also move portForwardingTests to  ItIstioMiiDomain.java as well because we don't want to import accesseWLSViaWLSUsingForwardedPort that is in ItIstioMiiDomain.java from CommonTestUtils.java